newIDE/app/src/Profile/ChangeEmailDialog.js

```diff
@@ -64,12 +64,10 @@ export default class ChangeEmailDialog extends Component<Props, State> {
       <Dialog
         title={<Trans>Change your email</Trans>}
         actions={actions}
-        onRequestClose={() => {
-          if (!changeEmailInProgress) onClose();
-        }}
-        onApply={this._onChangeEmail}
         maxWidth="sm"
-        cannotBeDismissed={true}
+        cannotBeDismissed={changeEmailInProgress}
+        onRequestClose={onClose}
+        onApply={this._onChangeEmail}
```

Comment:
Could it apply unwanted changes on dismiss?

Reply:
yep it could! But at the same time, if we have an option to apply when clicking outside, then... we'd rather ensure it's consistent everywhere?
(as in, if you can click outside, then it does what you configured it to do)